packages/editor-ui/src/modules/settings.ts

```diff
@@ -80,32 +80,34 @@ const module: Module<ISettingsState, IRootState> = {
 			context.commit('setPersonalizationAnswers', results);
 		},
 		async fetchPromptsData(context: ActionContext<ISettingsState, IRootState>) {
-			try {
-				const promptsData: IN8nPrompts = await getPromptsData(context.state.settings.instanceId);
+			if (context.rootGetters.isTelemetryEnabled) {
+				try {
```

Comment:
nitpick - early exit is clearer here
```suggestion
			if (!context.rootGetters.isTelemetryEnabled) {
			    return;
			}
	         	try {
```

Reply:
Simplified